### ISSUE TITLE
Looms no longer work

### ISSUE THREAD

**Issue by Sombrerian** [ViaVersion/ViaVersion#2938]
### `/viaversion dump` Output

https://dump.viaversion.com/a4aa8b5bcbeb25e06c3e9c2b1efa5ed70a3427c55d3a6f1065c1ac320f0e0877

### Console Error

```
Put the paste.gg link or text here.
```

### Bug Description

Looms no longer output anything.

1.18.2: https://prnt.sc/HrCO8xnTuYAo
1.19: https://prnt.sc/HbxGwYZSsD6-

### Steps to Reproduce

1. Place down a loom
2. Try changing banner colour

### Expected Behavior

Loom doesn't work.

### Checklist

- [X] Via plugins are only running on **EITHER** the backend servers (e.g. Paper) **OR** the proxy (e.g. BungeeCord), **not on both**.
- [X] I have included a ViaVersion dump.
- [X] If applicable, I have included a paste (**not a screenshot**) of the error.
- [X] I have tried the latest build(s) from https://ci.viaversion.com/ and the issue still persists.

**Comment by Barvalg** [comment]
![Platform: git--Paper--379%20%28MC%3A%201.18.2%29](https://img.shields.io/badge/Platform-git--Paper--379%20%28MC%3A%201.18.2%29-3498db?style=flat-square) </br>![ViaVersion (4.3.2-SNAPSHOT): 1 commits behind master](https://img.shields.io/badge/ViaVersion%20(4.3.2--SNAPSHOT)-1%20commit(s)%20behind%20master-yellow?style=flat-square) </br>![ViaBackwards(4.3.0): 2 commits behind master](https://img.shields.io/badge/ViaBackwards%20(4.3.0)-2%20commit(s)%20behind%20master-yellow?style=flat-square) </br>

**Comment by Kichura** [comment]
Managed to reproduce this problem aswell - Workaround for now is to use 1.18.2 and trying to get the items that way or using commands to assign the tags.

**Comment by radiant-ai** [comment]
Wanted to fill a bug report but found this one, good. 1.19 clients indeed cannot use loom on 1.18.2 servers. Not a big problem for me as I have a different system. But here is one more dump if needed:
https://dump.viaversion.com/fa864750034fec1a881297c4fd0763be4a87d7574c2bcdeaf07c922543fb5007

**Comment by Barvalg** [comment]
![Platform: git--Purpur--1632%20%28MC%3A%201.18.2%29](https://img.shields.io/badge/Platform-git--Purpur--1632%20%28MC%3A%201.18.2%29-3498db?style=flat-square) </br>![ViaVersion (4.3.2-SNAPSHOT): Even with master](https://img.shields.io/badge/ViaVersion%20(4.3.2--SNAPSHOT)-Even%20withmaster-brightgreen?style=flat-square) </br>![ViaBackwards(4.3.1-SNAPSHOT): 1 commits behind master](https://img.shields.io/badge/ViaBackwards%20(4.3.1--SNAPSHOT)-1%20commit(s)%20behind%20master-yellow?style=flat-square) </br><br>Please try the **latest** build(s) from [https://ci.viaversion.com/](https://ci.viaversion.com/)

**Comment by Boy0000** [comment]
Tried updating to the latest devbuild but still can confirm it happens.
Seems abit less often than on 4.3.1? Specifically there it crashed when shift clicking the item

https://user-images.githubusercontent.com/62521371/175721469-0b6eb81c-9173-4191-be8b-5fd67a203852.mp4

**Comment by spookymgmt** [comment]
Can confirm crash whenever using loom.

**Comment by buzzcoding** [comment]
This is probably related to [https://bugs.mojang.com/browse/MC-254349](https://bugs.mojang.com/browse/MC-254349), which was fixed in 1.19.1-pre6.

**Comment by Krakenied** [comment]
still doesn't work, for 1.19.1 as well

**Comment by spookymgmt** [comment]
Still doesn't work

**Comment by Thorkull** [comment]
Happening on our 1.18.2 server with 1.19.1 clients, as well.

**Comment by Krakenied** [comment]
Still broken

**Comment by Barvalg** [comment]
![Platform: git--Paper--387%20%28MC%3A%201.18.2%29](https://img.shields.io/badge/Platform-git--Paper--387%20%28MC%3A%201.18.2%29-3498db?style=flat-square) </br>![ViaVersion (4.6.3-SNAPSHOT): Even with master](https://img.shields.io/badge/ViaVersion%20(4.6.3--SNAPSHOT)-Even%20with%20master-brightgreen?style=flat-square) </br>![ViaBackwards(4.6.2-SNAPSHOT): Even with master](https://img.shields.io/badge/ViaBackwards%20(4.6.2--SNAPSHOT)-Even%20with%20master-brightgreen?style=flat-square) </br>

**Comment by Birsenberg** [comment]
Similar problem.

**Comment by koqel** [comment]
Yeah same here. Works on 1.18.2 but once you get on 1.19 the loom stuff is just invisible.

**Comment by DaJMaN4** [comment]
still broken

**Comment by Kichura** [comment]
Update: I have looked into this issue and turns out stonecutter is broken too.

**Comment by IranMine123** [comment]
Not Working. help
